src/renderer/components/watch-video-comments/watch-video-comments.js

```diff
@@ -319,5 +365,86 @@ export default defineComponent({
           this.isLoading = false
         })
     },
+
+    toggleCommentHighlight: async function (comment) {
+      if (this.isCommentHighlighted(comment)) {
+        showToast(this.$t('Unhighlighting comment'))
+        await this.$store.dispatch('unhighlightComment', { videoId: this.id, comment: comment })
```

Comment:
You already use `mapActions` at the bottom of the file, please use that for everything instead of doing some with that and others with `dispatch`.